assets/js/components/adminbar/AdminBarImpressions.js

```diff
@@ -88,27 +85,14 @@ const AdminBarImpressions = ( { className } ) => {
 	const totalImpressionsChange = calculateChange( totalOlderImpressions, totalImpressions );
 
 	return (
-		<div className={ classnames(
-			'mdc-layout-grid__cell',
-			className,
-		) }>
-			<DataBlock
-				className="overview-total-impressions"
-				title={ __( 'Total Impressions', 'google-site-kit' ) }
-				datapoint={ totalImpressions }
-				change={ totalImpressionsChange }
-				changeDataUnit="%"
-			/>
-		</div>
+		<DataBlock
+			className="overview-total-impressions"
+			title={ __( 'Total Impressions', 'google-site-kit' ) }
+			datapoint={ totalImpressions }
+			change={ totalImpressionsChange }
+			changeDataUnit="%"
+		/>
 	);
-};
-
-AdminBarImpressions.propTypes = {
-	className: PropTypes.string,
-};
-
-AdminBarImpressions.defaultProps = {
-	className: 'mdc-layout-grid__cell--span-2-tablet mdc-layout-grid__cell--span-3-desktop',
-};
+}
 
-export default AdminBarImpressions;
+export default withWidgetComponentProps( { widgetSlug: WIDGET_SLUG } )( AdminBarImpressions );
```

Comment:
See above, this should be removed from here, instead the raw component should be returned.